cli/jobs/train/r/README.md

```diff
@@ -0,0 +1,40 @@
+---
+page_type: sample
+languages:
+- R
+- azurecli
+products:
+- azure-machine-learning
+description: Learn how to submit R jobs to an AzureMLCompute Cluster using the AzureML 2.0 CLI.
+---
+
+# R using the AzureML 2.0 CLI
+
+This directory provides some sample projects to run R using the AzureML 2.0 CLI.
+
+## Environment
+
+All the projects in this directory use the `rocker/tidyverse:4.0.0-ubuntu18.04` image from Docker Hub. This image has the tidyverse and its dependencies installed (see [Rocker](https://github.com/rocker-org/rocker) for more details).
+
+In addition there are 2 things to point out about the [Dockerfile](./basic-train-model/Dockerfile) that builds the environment for these samples:
+
+In this section python is installed, since it is required by AzureML to start the job. This requirement will be removed for runs executed on cloud compute (AML Compute Cluster, AML Compute Instance) soon.
+```Dockerfile
+RUN apt-get update -qq && \
+ apt-get install -y python3-pip
+RUN ln -f /usr/bin/python3 /usr/bin/python
+RUN ln -f /usr/bin/pip3 /usr/bin/pip
+RUN pip install -U pip
+```
+
+In this next section, the dependencies for local execution with data mounting are installed. This is required if you are using cloud data in your job and you are running the job on the compute target `local` (i.e. your local docker runtime).
```

Comment:
from my testing, you actually don't need these dependencies installed locally, but rather in the job's environment when running locally